opendatahub-io/odh-dashboard
# Use ProjectRequests instead of Namespaces

Resolves: #732 

## Description

- Changed from namespaces to projectrequests
- Suffix and hide the display name of `[DSP]` -- this way we can still filter client side for projects created by us
    - This is inefficient from a fetching point of view -- but projects don't allow us to label without going through the Namespace object (which we can guarantee we will have permissions for)

## How Has This Been Tested?



- Create a project in the Dashboard, look at it in OpenShift Console and see the display name suffixed.
- Create a project in OpenShift Console with a suffixed display name of `[DSP]`, look at it in the Dashboard.

## Merge criteria:
